src/index.js

```diff
@@ -0,0 +1,89 @@
+import './style.css';
+import {
+  mousedown, dragndrop,
+} from './dragndrop';
+import { status, prepopstatus } from './status';
+import {
+  removecompleted, removeAll, Duty, edit, removetask,
+} from './task';
+
+const form = document.getElementById('form');
+const taskinput = document.querySelector('.taskadder');
+const sync = document.querySelector('.sync');
+const entericon = document.querySelector('.enter-icon');
+const deletecompleted = document.getElementById('delcompleted');
+const list = [];
+let displayedList;
+
+const todoList = (arr) => {
```

Comment:
Great Work. I found your code readable, and there are no many things I would suggest changing, anyways. My suggestion is, it will improve code readability a lot, which is the basic principle of code duty separation. Just group DOM manipulation in function and not mix logic and dom manipulation.

You can use string interpolation or separate code related to dom element creation and styling in another function than the call function. Of course, this is just an optional suggestion.

Great work, my friend!

Reply:
Thank you!